opencensus/stats/view_descriptor.h

```diff
@@ -82,6 +82,11 @@ class ViewDescriptor final {
   ViewDescriptor& set_description(absl::string_view description);
   const std::string& description() const { return description_; }
 
+  // Sets the desired lifetime of the view's data. If the view data is not
+  // updated at any point during this duration, it will be removed from the set
+  // of data recorded by the view when periodic aggregation happens.
+  ViewDescriptor& set_expiry_duration(const absl::Duration& expiry_duration);
```

Comment:
Please pass Duration by value, not by constref.

Reply:
Done.